### PR TITLE
Feature/1975 - Separation of Content and Code

### PR DESCRIPTION
**Pull request by davidjgonzalez**
This PR splits Content and Code into discrete sub-packages of the `acs-aem-commons content package`.

### CODE REVIEW

File: content/pom.xml [hunk 1]
```diff
@@ -161,157 +251,49 @@
...
+            <type>content-package</type>
         </dependency>
         <dependency>
-            <groupId>commons-collections</groupId>
-            <artifactId>commons-collections</artifactId>
-            <version>3.2.1</version>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>acs-aem-commons-ui.content</artifactId>
+            <version>${project.version}</version>
+            <type>content-package</type>
```
Comment (adamcin) [inline content/pom.xml]: I think the `type` value here should be `zip`.

File: content/pom.xml [hunk 2]
```diff
@@ -161,157 +251,49 @@
...
-            <version>3.0.1</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>acs-aem-commons-ui.content</artifactId>
+            <version>${project.version}</version>
+            <type>content-package</type>
```
Comment (adamcin) [inline content/pom.xml]: I think the `type` value here should be `zip`.

File: content/pom.xml [hunk 3]
```diff
@@ -161,157 +251,49 @@
...
     </build>
+
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>acs-aem-commons-bundle</artifactId>
+            <artifactId>acs-aem-commons-ui.apps</artifactId>
             <version>${project.version}</version>
-            <scope>provided</scope>
+            <type>content-package</type>
```
Comment (adamcin) [inline content/pom.xml]: I think the `type` value here should be `zip`.

File: content/pom.xml [hunk 4]
```diff
@@ -161,157 +251,49 @@
...
-            <groupId>com.day.cq.wcm</groupId>
-            <artifactId>cq-wcm-taglib</artifactId>
-            <version>5.6.4</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.scripting.jsp.taglib</artifactId>
-            <version>2.1.8</version>
+            <type>content-package</type>
```
Comment (adamcin) [inline content/pom.xml]: I think the `type` value here should be `zip`.

**Comment by davidjgonzalez** [comment]
@justinedelson @adamcin  we good to merge this now? (I fixed the comments from @adamcin)

**Comment by adamcin** [comment]
@davidjgonzalez I think you also need to change the coveralls task in `.travis.yml` to reach the `package` goal (because prepare-package doesn't build the `min` depedencies) or `verify` goal (for IT test coverage inclusion). see: https://github.com/davidjgonzalez/acs-aem-commons/pull/7/files#diff-354f30a63fb0907d4ad57269548329e3R14

**Comment by davidjgonzalez** [comment]
@adamcin ahh shoot - didnt even see that PR notification :( sorry - trying to do too many things at once. Merged it!

**Pull request merged**